common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/dialog/CustomAlertDialogBuilder.kt

```diff
@@ -151,4 +164,13 @@ class CustomAlertDialogBuilder(val context: Context) : DaxAlertDialog {
             }
         }
     }
+
+    private fun checkRequiredFieldsSet() {
+        if (positiveButtonText.isEmpty()) {
+            throw Exception("CustomAlertDialogBuilder: You must always provide a Positive Button")
+        }
+        if (negativeButtonText.isEmpty()) {
+            throw Exception("CustomAlertDialogBuilder: You must always provide a Negative Button")
+        }
```

Comment:
I removed this logic because we didn't want them to be mandatory as we could have a different custom view with different buttons